src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -43,26 +42,25 @@ internal abstract partial class CSharpSelectionResult : SelectionResult
             var newDocument = await SemanticDocument.CreateAsync(document.Document.WithSyntaxRoot(root.AddAnnotations(
                     new[]
                     {
-                        Tuple.Create<SyntaxToken, SyntaxAnnotation>(firstToken, firstTokenAnnotation),
-                        Tuple.Create<SyntaxToken, SyntaxAnnotation>(lastToken, lastTokenAnnotation)
+                        Tuple.Create(firstToken, firstTokenAnnotation),
+                        Tuple.Create(lastToken, lastTokenAnnotation)
```

Comment:
i'll likely move to using normal value tuples over reference tuples in this pr as well.